src/intl/translated-messages.json

```diff
@@ -305,8 +306,8 @@
     "diskEditorDiskTypeCreateHelp": "Sobald Sie einen Disk-Typ ausgewählt haben, können Sie ihn nicht mehr ändern.",
     "diskEditorDiskTypeLabel": "Disk-Typ",
     "diskEditorDiskTypeNotAvailable": "n. z.",
-    "diskEditorDiskTypeOptionThin": "Schlanke Speicherzuweisung",
     "diskEditorDiskTypeOptionPre": "Vorab zugeordnet",
+    "diskEditorDiskTypeOptionThin": "Schlanke Speicherzuweisung",
```

Comment:
it seems that in the new version the keys are now sorted correctly. How it's possible that previously 
the sort order was wrong? do we have some bug in our zanata scripts?